server/issue_parser.go

```diff
@@ -34,7 +34,45 @@ func reporterSummary(issue *jira.Issue) string {
 	return reporterSummary
 }
 
-func parseIssue(issue *jira.Issue) []*model.SlackAttachment {
+func getTransitionActions(client Client, issue *jira.Issue) ([]*model.PostAction, error) {
+	var actions []*model.PostAction
+
+	ctx := map[string]interface{}{
+		"issueKey": issue.ID,
+	}
+
+	integration := &model.PostActionIntegration{
+		URL:     fmt.Sprintf("/plugins/%s%s", manifest.Id, routeIssueTransition),
+		Context: ctx,
+	}
+
+	var options []*model.PostActionOptions
+
+	transitions, err := client.GetTransitions(issue.Key)
+	if err != nil {
+		return actions, err
+	}
+
+	for _, transition := range transitions {
+		if transition.Name != issue.Fields.Status.Name {
+			options = append(options, &model.PostActionOptions{
+				Text:  transition.Name,
+				Value: transition.Name,
```

Comment:
Can we use `transition.ID` here for the `Value`, and convert `transitionJiraIssue` to use the `ID` instead of trying to match the `Name`?

We originally had it this way so the user could use a command `/jira transition (key) Resolved`, but this feature is essentially replacing the `/jira transition` command. cc @aaronrothschild

Reply:
Hello @mickmister wouldn't this be better done in another ticket after `transitionJiraIssue` is converted to use transition `ID` instead?

Reply:
Yes, sounds good to me :+1: